.swiftformat

```diff
@@ -0,0 +1,106 @@
+# Loosely based on SwiftFormat config compliant with Google Swift Guideline
+# https://google.github.io/swift/#control-flow-statements
+# https://github.com/NoemiRozpara/Google-SwiftFormat-Config
+
+# Specify version used in a project
+
+--swiftversion 5.1
+
+# Rules explicitly required by the guideline
+
+--rules                           \
+blankLinesAroundMark,             \
+blankLinesAtEndOfScope,           \
+blankLinesAtStartOfScope,         \
+blankLinesBetweenScopes,          \
+braces,                           \
+consecutiveBlankLines,            \
+consecutiveSpaces,                \
+duplicateImports,                 \
+elseOnSameLine,                   \
+emptyBraces,                      \
+enumNamespaces,                   \
+extensionAccessControl,           \
+hoistPatternLet,                  \
+indent,                           \
+leadingDelimiters,                \
+linebreakAtEndOfFile,             \
+markTypes,                        \
+organizeDeclarations,             \
+redundantInit,                    \
+redundantParens,                  \
+redundantPattern,                 \
+redundantRawValues,               \
+redundantType,                    \
+redundantVoidReturnType,          \
+semicolons,                       \
+sortedImports,                    \
+spaceAroundBraces,                \
+spaceAroundBrackets,              \
+spaceAroundComments,              \
+spaceAroundGenerics,              \
+spaceAroundOperators,             \
+spaceAroundParens,                \
+spaceInsideBraces,                \
+spaceInsideBrackets,              \
+spaceInsideComments,              \
+spaceInsideGenerics,              \
+spaceInsideParens,                \
+todos,                            \
+trailingClosures,                 \
+trailingCommas,                   \
+trailingSpace,                    \
+typeSugar,                        \
+void,                             \
+wrap,                             \
+wrapArguments,                    \
+wrapAttributes,                   \
+#
+#
+# Additional rules not mentioned in the guideline, but helping to keep the codebase clean 
+# Quoting the guideline: 
+# Common themes among the rules in this section are: 
+# avoid redundancy, avoid ambiguity, and prefer implicitness over explicitness
+# unless being explicit improves readability and/or reduces ambiguity.
+#
+#
+andOperator,                      \
+isEmpty,                          \
+redundantBackticks,               \
+redundantBreak,                   \
+redundantExtensionACL,            \
+redundantGet,                     \
+redundantLetError,                \
+redundantNilInit,                 \
+redundantObjc,                    \
+redundantReturn,                  \
+redundantSelf,                    \
+strongifiedSelf
+
+
+# Options for basic rules
+
+--extensionacl on-declarations
+--funcattributes prev-line
+--indent 4
+--ifdef no-indent
+--self remove
+--maxwidth 100
```

Comment:
nit: typically I don't break long lines (I have a wide monitor). Personal preference, but if all agree we can remove this.